rmgpy/data/kinetics/family.py

```diff
@@ -258,6 +258,7 @@ def get_reverse(self):
                 other.add_action(['GAIN_PAIR', action[1], action[2]])
             elif action[0] == 'GAIN_PAIR':
                 other.add_action(['LOSE_PAIR', action[1], action[2]])
+        other.actions = reversed(other.actions) # Play the reverse recipe in the reverse order
```

Comment:
`reversed` returns an iterator. Do we expect any issues with that?

Reply:
I didn't think that would cause any issues.  I'm able to run it perfectly fine.  Maybe @rwest feels differently?

Reply:
Hard to anticipate, but might as well try `other.actions = list(reversed(other.actions))`.

Then should (rebase and) look into why the unit tests failed.

Reply:
Actually, I have a better suggestion: use reversed directly on the for loop, `for action in reversed(self.actions)`